src/components/screenreader-only/screenreader-only.ts

```diff
@@ -0,0 +1,26 @@
+import type { CSSResultGroup, TemplateResult } from 'lit';
+import { html, LitElement } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+import style from './screenreader-only.scss?lit&inline';
+
+/**
+ * This component can be used to visually hide content but present it to screen readers.
+ *
+ * @slot - Use the unnamed slot to provide content.
+ */
+@customElement('sbb-screenreader-only')
+export class SbbScreenreaderOnlyElement extends LitElement {
+  public static override styles: CSSResultGroup = style;
+
+  protected override render(): TemplateResult {
+    return html`<span><slot></slot></span>`;
```

Comment:
I would vote to keep this completely flat, without even the span. This is an internal component for a very specific use case. If someone breaks it, its completely their own fault.